tests/components/zerproc/test_light.py

```diff
@@ -44,7 +44,7 @@ async def mock_light(hass, mock_entry):
     light = MagicMock(spec=pyzerproc.Light)
     light.address = "AA:BB:CC:DD:EE:FF"
     light.name = "LEDBlue-CCDDEEFF"
-    light.connected = False
+    light.is_connected.return_value = False
```

Comment:
Perhaps `mock_light` is unused and can just be removed, even? It seems like each test is creating their own MagicMoc.

Reply:
The `mock_light` fixture is still used in the tests below

Reply:
Apologies, I think I just missed the hidden parts of the diff!

Reply:
No worries! I had to go back and look myself to figure out what was going on.